utils/resultstable.go

```diff
@@ -466,6 +466,7 @@ func prepareSast(sasts []*sarif.Run, isTable bool) []formats.SourceCodeRow {
 						SeverityDetails:    severityutils.GetAsDetails(currSeverity, jasutils.Applicable, isTable),
 						ScannerDescription: scannerDescription,
 						Finding:            sarifutils.GetResultMsgText(sastResult),
+						Fingerprint:        sarifutils.GetResultFingerprint(sastResult),
```

Comment:
add it also for `Secrets` and `Iac` if not exist for full support. it will just be set empty and omitted. 
if one day they will also add it to the other scanners it will just fill up the values without additional changes.